src/storage/v3/request_helper.cpp

```diff
@@ -438,6 +444,61 @@ ShardResult<msgs::ExpandOneResultRow> GetExpandOneResult(
   return result_row;
 }
 
+std::vector<GetPropElement> OrderByElements(DbAccessor &dba, std::vector<msgs::OrderBy> &order_by,
+                                            std::vector<GetPropElement> &&vertices) {
+  std::vector<Ordering> ordering;
+  ordering.reserve(order_by.size());
+  for (const auto &order : order_by) {
+    switch (order.direction) {
+      case memgraph::msgs::OrderingDirection::ASCENDING: {
+        ordering.push_back(Ordering::ASC);
+        break;
+      }
+      case memgraph::msgs::OrderingDirection::DESCENDING: {
+        ordering.push_back(Ordering::DESC);
+        break;
+      }
+    }
+  }
+  struct PropElement {
+    std::vector<TypedValue> properties_order_by;
```

Comment:
These are not properties. These are expressions that evaluated on the vertices/edges, so this was confusing for me.